modules/ecr__enum/main.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+import argparse
+import json
+from copy import deepcopy
+from botocore.exceptions import ClientError
+import time
+
+module_info = {
+    'name': 'ecr__enum',
+    'author': 'Manas Bellani',
+    'category': 'ENUM',
+    'one_liner': 'Enumerates repostories and relevant images/tags ',
+    'description': 'This module enumerates information about all ECR images and repositories within Elastic Container Registry (ECR). It writes the JSON results found to "Downloads" folder within sessions folder',
+    'services': ['ECR'],
```

Comment:
This line needs to be `'services': ['API.ECR'],` otherwise it does not match any service resulting in:

```
  File "pacu.py", line 985, in list_modules
    regions += self.get_regions(service)
  File "pacu.py", line 221, in get_regions
    if type(regions[service]) == dict and regions[service].get('endpoints'):
<class 'KeyError'>: 'ecr'
```